src/docs/product/reference/search/searchable-properties/user-feedback.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: User Feedback Properties
+sidebar_order: 40
+description: "Learn more about searchable user feedback properties."
+---
+
+<Include name="searchable-properties/intro-for-user-feedback.mdx" />
+
+## Searchable Properties
+
+Below is a list of keys and tokens that can be used in the user feedback search.
+
+<DefinitionList>
+
+### `assigned`
+
+Returns issues assigned to the defined user(s) or team(s). Values can be a user ID (your email address), `me` for yourself, `none` for no assignee, `my_teams` or `#team-name` for teams you belong to.
+
+- **Type:** team or org user
+
+### `browser.name`
+
+Name of the browser.
+
+- **Type:** string
+
+### `device.brand`
+
+Brand of the device.
+
+- **Type:** string
+
+### `device.family`
+
+Family of the device. Typically, the common part of a model name across generations. For example, iPhone, Samsung Galaxy.
+
+- **Type:** string
+
+### `device.model_id`
+
+Internal hardware revision to identify the device exactly.
+
+- **Type:** n/a
+
+### `device.name`
+
+Name of the device.
+
+- **Type:** string
+
+### `dist`
+
+Distinguishes build or deployment variants of the same release of an application. For example, the dist can be the build number of an XCode build or the version code of an Android build.
+
+- **Type:** string
+
+### `environment`
+
+The environment that the event was first seen in.
+
+- **Type:** string
+
+### `id`
+
+The feedback ID.
+
+- **Type:** UUID
+
+### `is`
+
+The properties of an issue. Values can be: `unresolved`, `resolved`, `assigned`, `unassigned`, `linked`, or `unlinked`. The `linked` and `unlinked` values return issues based on whether they are linked to an external issue tracker or not.
+
+- **Type:** status
+
+### `level`
+
+The severity of the user feedback. If the feedback came in through the crash-report modal, then the level would be `error`; otherwise, the level is `info`.
```

Comment:
```suggestion
The severity of the user feedback for submissions that came in through the crash-report modal and have an error attached.
```

Reply:
> otherwise, the level is `info`.
-
Do we reflect anywhere that the feedback submission from our user feedback widget is "info" level? I don't think that's part of the UI anymore :/ Perhaps we can still leave the note that submissions from the user feedback widget should have an info level in order for people to filter their results for just that.

Reply:
nope, we don't indicate the "info" level anywhere

Reply:
Ok I suggested something else!